### PR TITLE
Feature/pdp

### PR DESCRIPTION
**Pull request by lusiakpurnama**

### CODE REVIEW

File: docs/components/product-image-carousel.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+<body class="mobile-wrapper">
+	<div class="container mt-4">
+		<h1>Product Image Carousel</h1>
+
+		<div class="row">
+			<div class="product-image-carousel__container col-12 col-lg-6 order-lg-2">
+				<figure id="product-image-carousel" class="product-image-carousel carousel slide mb-1 no-gutters__in-container" data-ride="carousel"  data-interval="false">
+					<div class="carousel-inner">
+						<div class="carousel-item active">
+							<img class="img-fluid" src="https://via.placeholder.com/750x750/EFADBA?text=1"
```
Comment (foria) [inline docs/components/product-image-carousel.html:22]: can we just use image as carousel-item?

File: docs/templates/product.html [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+				</ul>
+
+			</div>
+		</div>
+	</div>
+
+	<div class="container-fluid">
+		<div class="row">
+			<div class="col-12 col-lg-6 px-g px-lg-0 d-lg-flex align-items-center" style="background-color: #F9EECA7f;">
+				<div class="pdp-banner__content ml-lg-auto px-lg-g pt-4 pb-2 py-lg-0">
```
Comment (foria) [inline docs/templates/product.html:261]: we don't have a pdp-banner so pdp-banner__content can be just pdp-banner

File: src/scss/components/_list.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.list-check {
+	li {}
```
Comment (foria) [inline src/scss/components/_list.scss:2]: please remove empty selectors

File: src/scss/core/_grid.scss [hunk 4]
```diff
@@ -6,7 +6,7 @@
 }
 
 @include media-breakpoint-up(lg) {
-	.container {
+	.container, .container-fluid {
```
Comment (foria) [inline src/scss/core/_grid.scss:9]: why do we need padding on container-fluid? in this way it's not gonna be full-width

**Review by foria — CHANGES_REQUESTED** [review]

File: docs/components/product-image-carousel.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+							<img class="img-fluid" src="https://via.placeholder.com/750x750/EFADBA?text=8"
+								alt="Slide 8">
+						</div>
+					</div>
+				</figure>
+			</div>
+			
+			<div id="product-image-carousel__indicator" class="product-image-carousel__indicator col-12 col-lg-1 order-lg-1">
+				<div class="row">
+					<div class="product-image-carousel__indicator__item active col-12 mb-1" data-target="#product-image-carousel" data-index="1">
```
[the remark this answers is not in the document]
Comment (lusiakpurnama) [reply docs/components/product-image-carousel.html:59]: seems its an old committed file, doenst have this in the latest one, let me update

File: docs/components/product-image-carousel.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+	<link rel="stylesheet" href="/cocoandeve-styleguides/dist/css/main.css">
+</head>
+
+<body class="mobile-wrapper">
+	<div class="container mt-4">
+		<h1>Product Image Carousel</h1>
+
+		<div class="row">
+			<div class="product-image-carousel__container col-12 col-lg-6 order-lg-2">
+				<figure id="product-image-carousel" class="product-image-carousel carousel slide mb-1 no-gutters__in-container" data-ride="carousel"  data-interval="false">
```
[the remark this answers is not in the document]
Comment (lusiakpurnama) [reply docs/components/product-image-carousel.html:19]: i cannot get col class work together with the no-gutters__in-container class

File: docs/templates/product.html (hunk 2 above)
Comment (lusiakpurnama) [reply docs/templates/product.html:261]: well, my intention is that those 2 content is a part of 1 banner, so i added `__content`. That class is actually to make half a container. Should i just rename it to `.container__half`?

File: src/scss/core/_grid.scss (hunk 4 above)
Comment (lusiakpurnama) [reply src/scss/core/_grid.scss:9]: actually, `.container-fluid` also have gutter padding. What I did here is to adjust it to have desktop gutter with in lg screen.

**Pull request merged**